### PR TITLE
[ISSUE #6235] Some minor code optimizations 

### LINKED ISSUE #6235: Removed the ForwardRequestProcessors class that is not being used
org.apache.rocketmq.broker.processor.ForwardRequestProcessor in module Broker, does not appear to be used.
Would like to initiate a discussion to see if it can be removed.

Removed the ForwardRequestProcessors class that is not being used

### PR DESCRIPTION
**Pull request by socutes**
close #6235

### CODE REVIEW

File: broker/src/main/java/org/apache/rocketmq/broker/processor/AckMessageProcessor.java [hunk 1]
```diff
@@ -94,16 +94,15 @@ public boolean isPopReviveServiceRunning() {
...
     }
 
     @Override
     public boolean rejectRequest() {
         return false;
     }
 
-    private RemotingCommand processRequest(final Channel channel, RemotingCommand request,
-        boolean brokerAllowSuspend) throws RemotingCommandException {
+    private RemotingCommand processRequest(final Channel channel, RemotingCommand request) throws RemotingCommandException {
```
Comment (caigy) [inline broker/src/main/java/org/apache/rocketmq/broker/processor/AckMessageProcessor.java:105]: This param may be useful in the future, so pls communicate with the community before modify it.

File: broker/src/main/java/org/apache/rocketmq/broker/processor/AckMessageProcessor.java [hunk 2]
```diff
@@ -160,8 +159,7 @@ private RemotingCommand processRequest(final Channel channel, RemotingCommand re
             if (requestHeader.getOffset() < oldOffset) {
                 return response;
             }
-            while (!this.brokerController.getPopMessageProcessor().getQueueLockManager().tryLock(lockKey)) {
-            }
+
```
Comment (caigy) [inline broker/src/main/java/org/apache/rocketmq/broker/processor/AckMessageProcessor.java:162]: Could you explain the reason to remove the locking statement?

File: broker/src/main/java/org/apache/rocketmq/broker/processor/ForwardRequestProcessor.java [hunk 3]
```diff
@@ -1,45 +0,0 @@
-/*
```
Comment (caigy) [inline broker/src/main/java/org/apache/rocketmq/broker/processor/ForwardRequestProcessor.java:1]: Have you launched a discuss about removing this class in community?

**Review by caigy — CHANGES_REQUESTED** [review]

**Comment by socutes** [comment]
@caigy cai Sorry, I did not consider this requirement well, there is a problem with this PR, I will close it

**Pull request closed without merging**